src/Handler/Customer/SendResetPasswordTokenHandler.php

```
@@ -31,14 +31,15 @@ public function __invoke(SendResetPasswordToken $resendResetPasswordToken): void
 
         /** @var ShopUserInterface $user */
         $user = $this->userRepository->findOneByEmail($email);
-
-        Assert::notNull($user, sprintf('User with %s email has not been found.', $email));
```

The handler should throw an exception anyway, I suppose.

---

Check Now